sdk/include/psdk/shellapi.h

```diff
@@ -48,6 +48,8 @@ extern "C" {
 #define SEE_MASK_HASLINKNAME	0x00010000
 #define SEE_MASK_HASTITLE	0x00020000
 #define SEE_MASK_FLAG_SEPVDM	0x00040000
+#define SEE_MASK_UNKNOWN_0x1000 0x1000 /* FIXME: Name */
+#define SEE_MASK_HASCLASS 0x80000 /* FIXME: Name */
```

Comment:
`SEE_MASK_HASCLASS` cannot be correct because `SEE_MASK_CLASSNAME` is documented and has a different value. The flag must be for something else like .Lnk path? Title?

Reply:
Renamed as `SEE_MASK_HASTITLENEW`.